src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -46,8 +46,10 @@ public void AddToMempool()
                 CoreNode stratisNodeSync = builder.CreateStratisPowNode(this.network);
                 builder.StartAll();
 
+                stratisNodeSync.FullNode.Network.Consensus.CoinbaseMaturity = 1L;
+
                 stratisNodeSync.SetDummyMinerSecret(new BitcoinSecret(new Key(), stratisNodeSync.FullNode.Network));
-                stratisNodeSync.GenerateStratisWithMiner(105); // coinbase maturity = 100
+                stratisNodeSync.GenerateStratisWithMiner(5);
```

Comment:
maybe create `int maturity = 5` and then do `stratisNodeSync.GenerateStratisWithMiner(maturity + 5);` so that the calculation is clear?

Reply:
could you not change the network maturity in the ctor? then it'll be available for the whole class.